tests/seed_isort_config_test.py

```diff
@@ -295,7 +309,7 @@ def test_exclude(tmpdir):
 def test_returns_zero_no_changes(tmpdir):
     with tmpdir.as_cwd():
         cfg = tmpdir.join('.isort.cfg')
-        cfg.write('[settings]\nknown_third_party=cfgv\n')
+        cfg.write(b'[settings]\nknown_third_party=cfgv\n')
```

Comment:
these shouldn't change (same for the second one)

Reply:
Fixed